inst/rmd/qc_report.rmd

```diff
@@ -381,6 +385,65 @@ If the model has failed to fit properly, the pattern of cells may differ, and th
 This can be the result of a low-quality library or may occur if there is no mitochondrial content, as in the case of a high-quality single-nucleus sample.
 In such situations, the calculated probability of compromise may not be valid (see [miQC vignette](https://bioconductor.org/packages/3.13/bioc/vignettes/miQC/inst/doc/miQC.html#when-not-to-use-miqc) for more details).
 
+## Removing low quality cells
+
+The below plot highlights cells that were removed prior to normalization and dimensionality reduction.
+Cells that should be removed are those that are identified to be low quality cells, such as cells with high probability of being compromised, calculated by `miQC`.
+The method of filtering is indicated above the plot as either `miQC` or `Minimum gene cutoff`. 
+If `miQC`, cells below the specified probability compromised cutoff and above the minimum number of unique genes identified are kept for downstream analyses. 
+If only a `Minimum gene cutoff` is used, then `miQC` is not used and only those cells that pass the minimum number of unique genes identified threshold are retained. 
+The dotted vertical line indicates the minimum gene cutoff used for filtering. 
+
+```{r}
+if(has_filtered & has_processed){
+  
+  # grab cutoffs and filtering method from processed sce
+  min_gene_cutoff <- processed_meta$min_gene_cutoff
+  prob_compromised_cutoff <- processed_meta$prob_compromised_cutoff
+  filter_method <- processed_meta$ccdl_filter_method
+  
+  # add column to coldata labeling cells to keep/remove based on filtering method
+  if(filter_method == "miQC"){
+    
+    filtered_coldata_df <- filtered_coldata_df %>%
+      dplyr::mutate(ccdl_filter = ifelse(prob_compromised <= prob_compromised_cutoff & 
+                                           detected >= min_gene_cutoff, 
+                                         "Keep", 
+                                         "Remove"))
+    
+  } else if(filter_method == "Minimum_gene_cutoff") {
+    
+    filtered_coldata_df <- filtered_coldata_df %>%
+      dplyr::mutate(ccdl_filter = ifelse(detected >= min_gene_cutoff, 
+                                         "Keep", 
+                                         "Remove"))
+    
+  }
```

Comment:
I think we want to simplify this to use the processed data. This also means we don't need different code for miQC & the simpler cutoff. 

We can probably move making `filtered_coldata_df` to here as well, since we won't need the `prob_compromised` at all
```suggestion
  filtered_coldata_df <- colData(filtered_sce) |>
    as.data.frame()
    
  filtered_coldata_df <- filtered_coldata_df |>
    dplyr::mutate(ccdl_filter = ifelse(rownames(filtered_coldata_df) %in%  colnames(processed_sce), 
                                       "Keep", 
                                       "Remove")) 
```